src/routes/cloud/[...id]/metrics/+page.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+	import { Table } from 'sveltestrap';
+	import MetricImplementationBlock from '$lib/MetricImplementationBlock.svelte';
+	import MetricConfigurationBlock from '$lib/MetricConfigurationBlock.svelte';
+	import { getMetricImplementation, getMetricConfiguration } from '$lib/orchestrator';
+	import type { Metric, MetricImplementation, MetricConfiguration} from '$lib/assessment';
+	import type { PageData } from './$types';
+
+	export let data: PageData;
+	
+	async function fetchImplementation(metric: Metric): Promise<MetricImplementation> {
+		return await getMetricImplementation(metric.id);
+	}
+
+	async function fetchConfiguration(cloud_service_id: string, metric: Metric): Promise<MetricConfiguration> {
```

Comment:
What is the idea behind that? you should be able to just call `getMetricConfiguration` and `getMetricImplementation` in the code below.